presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -2,10 +2,23 @@
 JDBC Driver
 ===========
 
-Presto can be accessed from Java using the JDBC driver.
-Download :maven_download:`jdbc` and add it to the class path of your Java application.
+The Presto JDBC (Java Database Connectivity) driver allows users to access 
+Presto using Java-based applications. Both client- and server-side 
+applications, such as those used for reporting and database development, use 
+the JDBC driver. The JDBC driver is an API that provides methods to perform 
+database operations such as insert, update, delete and query.
 
-The driver is also available from Maven Central:
+Requirements
+------------
+
+The JDBC driver is compatible with Java 8, and can be used with applications 
+running on Java virtual machines version 8 and higher.
+
+Installing
+----------
+
+Download the Presto :maven_download:`jdbc` or alternatively, you can download 
```

Comment:
I like the original text. It says

> Download presto-jdbc-340.jar

The proposed change seems redundant

> Download the Presto presto-jdbc-340.jar

Also, the original text "The driver is also available from Maven Central" seems better, since users don't really download something from Maven Central. Rather, they add a reference to it in their project and the dependency is automatically managed by Maven.